index.src.html

```diff
@@ -487,6 +488,11 @@ <h5 id="algorithm-create-cred" algorithm>`[[Create]]` internal method</h5>
 
         Note: This function was previously called `requireUserMediation()` which should be considered
         deprecated.
+
+    :   <dfn attribute>conditionalMediationSupported</dfn>
```

Comment:
Conditional mediation may be supported for one kind of credential but not for another. How are we planning to solve this? Perhaps it's okay to say that if `true`, the UA supports this on a best-effort basis depending on the credential. In practice this would mean for `password` credentials, it will be equivalent to `required`.

Reply:
Given the suggestion below regarding renaming this boolean and making it a method, perhaps we could go further and have it either (a) take a cred type as an argument and return true/false with regards to that cred type, or (b) return a list of cred types supporting conditional mediation...?